graph/api/src/main/java/org/jboss/windup/graph/model/resource/FileModel.java

```diff
@@ -134,14 +134,14 @@
     /**
      * Gets the ProjectModel that this file is a part of
      */
-    @Adjacency(label = FILE_TO_PROJECT_MODEL, direction = Direction.OUT)
-    ProjectModel getProjectModel();
+    @Adjacency(label = BOUND_PROJECT, direction = Direction.OUT)
+    ProjectModel getBoundProject();
```

Comment:
Bound project is a project to which this file is bound. As you explained to me earlier, this relation is rather vague and used differently for different situations. Yet there are more file to project relations. We should name them by their meaning, otherwise we could end up with getProject1(), getProject2(), getProject3(). It's like calling a variable "integer" and the getter "getInteger()".